components/client/src/main/java/com/hotels/styx/client/connectionpool/CloseAfterUseConnectionDestination.java

```diff
@@ -126,6 +128,13 @@ public void connectionClosed(Connection connection) {
         private Connection.Settings connectionSettings = new ConnectionSettings(1000, 1000);
         private Connection.Factory connectionFactory;
         private TlsSettings tlsSettings;
+        private HttpRequestOperationFactory requestOperationFactory = request -> new HttpRequestOperation(
```

Comment:
Do these fields not need to be configured in the factory?

Reply:
So this line was moved from SimpleNettyHttpClient.Builder, that was doing exactly the same type of initialisation.